### PR TITLE
Run Prettier on JS code fences, part 12

### PR DESCRIPTION
**Pull request by OnkarRuikar**
The PR focuses only on JS code fences.

Idea is to gradually prettify all the JS code fences before the full automation.

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/regexp/dotall/index.md [hunk 1]
```diff
@@ -36,22 +36,22 @@ The set accessor of `dotAll` is `undefined`. You cannot change this property dir
...
+const str2 = "bar\nexample foo example";
 
 const regex2 = /bar.example/;
 
 console.log(regex2.dotAll); // Output: false
 
-console.log(str2.replace(regex2,'')); // Output: bar
-                                      //         example foo example
+console.log(str2.replace(regex2, "")); // Output: bar
+//         example foo example
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/regexp/dotall/index.md:54]:
```suggestion
console.log(str2.replace(regex2, ""));
// bar
// example foo example
```

File: files/en-us/web/javascript/reference/global_objects/regexp/flags/index.md [hunk 2]
```diff
@@ -32,8 +32,8 @@ The set accessor of `flags` is `undefined`. You cannot change this property dire
 ### Using flags
 
 ```js
-/foo/ig.flags;   // "gi"
-/bar/myu.flags;  // "muy"
+/foo/gi.flags; // "gi"
+/bar/muy.flags; // "muy"
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/regexp/flags/index.md:36]: This was intentionally unordered. Maybe `nolint`?

File: files/en-us/web/javascript/reference/global_objects/regexp/source/index.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ Conceptually, the `source` property is the text between the two forward slashes
 ### Using source
 
 ```js
-const regex = /fooBar/ig;
+const regex = /fooBar/gi;
 
 console.log(regex.source); // "fooBar", doesn't contain /.../ and "ig".
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/regexp/source/index.md:31]:
```suggestion
console.log(regex.source); // "fooBar", doesn't contain /.../ and "gi".
```

File: files/en-us/web/javascript/reference/global_objects/regexp/flags/index.md (hunk 2 above)
Comment (OnkarRuikar) [reply files/en-us/web/javascript/reference/global_objects/regexp/flags/index.md]: done

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**